### DISCUSSION TITLE
Taro转H5，原本写的标签样式，view，text等等，H5并没有，有什么好的办法可以解决？

### DISCUSSION THREAD

**Discussion by zhulbin** [NervJS/taro#13864] [Q & A]
原本Taro转微信小程序，现有需求需要转H5，在Taro转H5的过程中，原本项目中书写的标签样式，view, text, image等标签样式，在H5并没有，有没有什么好的方法可以将所有的标签样式有效的转换成H5所需要的样式。

**Comment by ZakaryCode** [comment]
直接写就行，h5 会将标签通过 [postcss-html-transform](https://nervjs.github.io/taro-docs/docs/config-detail#h5postcsshtmltransform) 转成对应的，例如：`text => taro-text-core`

**Reply by zhulbin** [reply]
就只能通过一个一个更改标签样式去更改嘛？
例如原本是 image {}，然后现在因为转了h5，需要写的样式只能是image, taro-image-core {}，目的去兼容h5的样式

**Reply by ZakaryCode** [reply]
不用改，v3.5+ 默认就有
